.github/workflows/integration-tests.yml

```diff
@@ -45,3 +43,10 @@ jobs:
         Y_SWEET_S3_BUCKET_PREFIX: testing
         Y_SWEET_S3_BUCKET_NAME: ysweet-testing-y-sweet-data
       working-directory: tests
+      timeout-minutes: 10
+
+    - uses: actions/upload-artifact@v3
```

Comment:
this PR adds log redirection to files for debugging (only for the native server, currently). This uploads those to GitHub to make debugging easier.